package.json

```diff
@@ -26,9 +26,10 @@
     "ember-cli-inject-live-reload": "^1.3.0",
     "ember-cli-qunit": "0.3.13",
     "ember-cli-uglify": "^1.0.1",
+    "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-legacy-views": "0.2.0",
```

Comment:
Sorry, it seems that npm reordered my devDependencies. I didn't remove `ember-disable-prototype-extensions', but rather it just moved up to line 29. I apologize for this bad diff.

You have much more experience with developing add-ons than me. How would one make ember-legacy-views part of the project? Does it just need to be copied into dependencies? (currently only in devDependencies) Feel free to just close this PR and do this upgrade to 1.13 yourself if that will be easier.